website/js/index.js

```diff
@@ -466,6 +575,26 @@ var VariantDetail = React.createClass({
                 </Row>
                 <Row>
                     <Col md={8} mdOffset={2}>
+                        { /* eslint-disable dot-notation */ }
+                        <h3>{variant["HGVS_cDNA"]}</h3>
```

Comment:
I see these everywhere: `/* eslint-disable dot-notation */`

Is there a reason to write it as `variant["HGVS_cDNA"]` and disable the linter rather than `variant.HGVS_cDNA` as the linter wants?